index.js

```diff
@@ -48,7 +48,7 @@ exports.ext = {
  */
 
 function Library (name, mode) {
-  if (!(this instanceof Library)) return new Library(name);
+  if (!(this instanceof Library)) return new Library(name, mode);
```

Comment:
great catch! I really need to add some `mode` tests... but I'm not totally sure how to "verify" that a library has been opened via a specific mode.